cli.py

```diff
@@ -45,15 +50,21 @@ def submit_biosamples(self, url, domain, aap_url, aap_username, aap_password):
         biosamples_count = 0
         for row in self.excel.data.values():
             if 'sample' in row:
+                sample = row['sample']
                 try:
-                    row['sample']['request'] = biosamples.encode_sample(row['sample'])
-                    row['sample']['biosample'] = biosamples.send_sample(row['sample']['request'])
-                    row['sample'].pop('request')
+                    sample['request'] = biosamples.encode_sample(sample)
+                    sample['biosample'] = biosamples.send_sample(sample['request'])
+                    sample.pop('request')
+                    if 'accession' in sample['biosample']:
+                        sample['study_accession'] = sample['biosample']['accession']
```

Comment:
Is `study_accession` a typo? Is it not a sample's accession?

Reply:
good spot!